applications/FSIapplication/custom_utilities/mvqn_recursive_convergence_accelerator.hpp

```diff
@@ -14,17 +14,14 @@
 /* System includes */
 
 /* External includes */
-//~ #include "utilities/math_utils.h"
-//~ #include <cmath>
-//~ #include <numeric>
 
 /* Project includes */
 #include "includes/define.h"
 #include "includes/variables.h"
 #include "includes/ublas_interface.h"
 #include "convergence_accelerator.hpp"
 
-#include "custom_utilities/qr_utility.h"            //QR decomposition utility used in matrix inversion.
+#include "../FSIapplication/custom_utilities/qr_utility.h"            //QR decomposition utility used in matrix inversion.
```

Comment:
the ../FSIapplication should not be needed.

also i think that the qr_utility shall be moved to the core

Reply:
Done. The QR utility has been moved to the core.